ociinstaller/steampipeimage.go

```diff
@@ -186,12 +186,13 @@ func getPluginImageData(layers []ocispec.Descriptor) (*PluginImage, error) {
 	// get the binary plugin file info
 	for _, mediaType := range MediaTypeForPlatform("plugin") {
```

Comment:
this code is a bit opaque - at least out of context like this

Add a short comment to explain thew control flow

Reply:
done